src/Connect.ts

```diff
@@ -101,6 +97,8 @@ export function defineMediaExt(type: Type, params: Params) {
             console.warn('Unknown params type ' + type);
             break;
     }
+    // Fix mediaExt in removing the possible '.' prefix
+    Util.trimStart(params.mediaExt, '.');
```

Comment:
Maybe we can remove trimming here and put it in Util.getExtension?
The only possibility for mediaExt to not be trimmed here is that Util.getExtension() return the extension with a "."

Reply:
getExtension must returned the '.' it has been done intentonnally:
- match the nodejs version of other lib
- allow to write code to remove safely the extension: `fileName = fileName .slice(0, - Util.getExtension(fileName ).length)`
- allow as you say to detect an 'empty' extension (stays an extension part of the file)
- allow to concat with fileName or folderPath safely (without risk to change it to a filename!)
- allow to concat `Util.getBaseFile` + `Util.getExtension`
It was also like this that Mona was doing for `Filesystem::getExtension`, and how is working C++17 => https://en.cppreference.com/w/cpp/filesystem/path/extension

I take a lot of precaution about base function